packages/commonwealth/client/scripts/views/components/Comments/ArchiveMsg/ArchiveMsg.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { CWIcon } from 'views/components/component_kit/cw_icons/cw_icon';
+import moment from 'moment';
+import './ArchiveMsg.scss';
+
+type ArchiveMsgProps = {
+  archivedAt: moment.Moment | null;
+};
+
+export const ArchiveMsg = ({ archivedAt }: ArchiveMsgProps) => {
+  return (
+    <div className="ArchiveMsg">
+      <div className="archive-msg-container">
+        <CWIcon iconName="archiveTrayFilled" iconSize="small" />
```

Comment:
I suggest we use a single format (snake/kebab/camel etc) for classnames

Reply:
For now, icon names are camel-cased. We'll likely stick to this for the foreseeable future.